packages/cli/test/integration/lib/check-loopback-project.suite.js

```diff
@@ -73,6 +73,26 @@ module.exports = function suiteCheckLoopBackProject(generator) {
       /No `@loopback\/core` package found/,
     );
 
+    testCheckLoopBack(
+      'throws an error if "@loopback/core" is only a dev dependency',
+      {
+        devDependencies: {
+          '@loopback/core': coreVer,
+        },
+      },
+      /No `@loopback\/core` package found/,
+    );
+
+    testCheckLoopBack(
+      'throws an error if "@loopback/core" is only a peer dependency',
+      {
+        peerDependencies: {
+          '@loopback/core': coreVer,
+        },
+      },
+      /No `@loopback\/core` package found/,
+    );
+
```

Comment:
What't the differences between these two test cases and those 2 cases below(`'passes if "@loopback/core" is a dev dependency'` and `'passes if "@loopback/core" is a peer dependency'`)?

Reply:
For a package, we can declare regular and peer dependencies. These two tests make sure we allow `@loopback/*` in both cases.

Reply:
I think @agnes512 was asking about these two and the ones from line 242. 
There two test with default options (decision: abort), the next two are with options `command: update`

Reply:
The `command: 'update'` indicates the library is used with `lb4 update` command while others are part of any `lb4` command that expects a LB application project.

Reply:
I see, thanks for the explanation 👍